docker-entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/vegeta-server/bin/vegeta-server --ip=0.0.0.0 --port=3000
```

Comment:
I just create #35 where I will be pushing the change for defaults 0.0.0.0:80. With that pushed you can change entrypoint.sh to use the defaults args. 

Change this line to `/vegeta-server/bin/vegeta-server` only.